### PR TITLE
Create transform.py to transform data for comparison

### PR DESCRIPTION
**Pull request by Simeon-JAA**
Transform.py created which includes the following:

- Function to read extracted data
- Functions that clean and transform data to be consistent with db
- Saving transformed data to a new csv

Other additions include:

- Import transform functions into main.py

### CODE REVIEW

File: comparison_pipeline/transform.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    return pd.read_csv(file_path)
+
+
+def format_authors(authors: str) -> list[str]:
+    """Returns a list of authors if they exist and ['No Author'] if none detected"""
+
+    if authors.lower() == "nan":
+        return ["No Author"]
+
+    if authors[:3].lower() == "by ":
```
[the remark this answers is not in the document]
Comment (Simeon-JAA) [reply comparison_pipeline/transform.py:22]: There are no changes yet - we agreed to go with "No Author" for consistency across missing authors.
Did I miss something??

[the remark this answers is not in the document]
Comment (jGreeman) [reply comparison_pipeline/transform.py:22]: There are change on transform.py inside pipeline that handle a couple of edge cases, e.g. where 'By ' is missing in the author column but the names are there.

[the remark this answers is not in the document]
Comment (jGreeman) [reply comparison_pipeline/transform.py:22]: like 1 or 2 lines

[the remark this answers is not in the document]
Comment (Simeon-JAA) [reply comparison_pipeline/transform.py:22]: Yeah just cross checked and seen (it was the line below the highlighted ones)

[the remark this answers is not in the document]
Comment (jGreeman) [reply comparison_pipeline/transform.py:22]: oh cool, my b

**Pull request merged**